### PR TITLE
Added host volumes for Database service

### PR DESCRIPTION
**Pull request by Vj-L**
### Describe your changes :

I worked on this to add the host volumes for the Database

#
### Type of change :

- [x] Bug fix

### CODE REVIEW

File: docker/run_local_docker.sh [hunk 1]
```diff
@@ -67,8 +67,18 @@ fi
...
 docker compose  -f docker/local-metadata/docker-compose-postgres.yml down
 docker compose -f docker/local-metadata/docker-compose.yml down
+if [ -d "/docker-volume" ]
+then
+    echo "Do you want to delete the mounted directories from host?"
+    echo "Please enter [y/N]"
+    read  input
+    if [[ $input == "Y" || $input == "y" ]]; then
+      rm -rf $PWD/docker-volume
+    fi
```
Comment (akash-jain-10) [inline docker/run_local_docker.sh:77]: How will you provide the terminal input when running in CI ?

File: docker/run_local_docker.sh [hunk 2]
```diff
@@ -67,8 +67,18 @@ fi
...
+fi
 
 echo "Starting Local Docker Containers"
+mkdir docker-volume && mkdir docker-volume/db-data   && mkdir docker-volume/om-server
```
Comment (akash-jain-10) [inline docker/run_local_docker.sh:81]: Let's create docker volumes only for `db-data`. Also add the flag `-p` to mkdir command. ref - https://man7.org/linux/man-pages/man1/mkdir.1.html

File: ingestion/src/metadata/cli/docker.py [hunk 3]
```diff
@@ -65,12 +68,28 @@
 )
 
 
+def docker_volume():
+    # create a main directory
+    if not os.path.exists(MAIN_DIR):
+        os.mkdir(MAIN_DIR)
+        db = "db-data"
+        om_server = "om-server"
```
Comment (akash-jain-10) [inline ingestion/src/metadata/cli/docker.py:76]: Let's only create docker volumes for `db-data`.

**Review by akash-jain-10 — CHANGES_REQUESTED** [review]

File: ingestion/src/metadata/cli/docker.py (hunk 3 above)
Comment (Vj-L) [reply ingestion/src/metadata/cli/docker.py:76]: Done

File: docker/run_local_docker.sh (hunk 2 above)
Comment (Vj-L) [reply docker/run_local_docker.sh:81]: Done

File: docker/run_local_docker.sh (hunk 1 above)
Comment (Vj-L) [reply docker/run_local_docker.sh]: Done

**Review by chirag-madlani — APPROVED** [review]

**Review by akash-jain-10 — APPROVED** [review]

**Comment by harshach** [comment]
@vijaypm @akash-jain-10 what is remaining here. Lets follow-up and land it in quickly

**Review by akash-jain-10 — APPROVED** [review]

**Comment by akash-jain-10** [comment]
@Vj-L  - We should be good to merge the changes here along with documentation PR [here](https://github.com/open-metadata/OpenMetadata/pull/8788). Please go ahead with the merge. Thanks.

**Review by pmbrull — APPROVED** [review]

File: .github/workflows/cypress-integration-tests-mysql.yml [hunk 4]
```diff
@@ -130,3 +130,10 @@ jobs:
           # Recommended: pass the GitHub token lets this action correctly
           # determine the unique run id necessary to re-run the checks
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
```
Comment (ayush-shah) [inline .github/workflows/cypress-integration-tests-mysql.yml:133]: it's docker/local-metadata

Comment (Vj-L) [reply .github/workflows/cypress-integration-tests-mysql.yml:133]: Done 
Thanks

**Pull request merged**